ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh

```diff
@@ -115,6 +115,8 @@ function create_upi_tf_varfile(){
   export IBMCLOUD_API_KEY
   # OCP_TMP_VERSION=$(openshift-install version | grep openshift-install | awk '{print $2}')
   # echo "OCP_TMP_VERSION: ${OCP_TMP_VERSION}"
+  POWERVS_SERVICE_INSTANCE_ID="626e36c2-6a2e-4ce2-91e1-543185709880"
+  export POWERVS_SERVICE_INSTANCE_ID
 
   cat <<EOF >${IBMCLOUD_HOME_FOLDER}/ocp-install-dir/var-multi-arch-upi.tfvars
 ibmcloud_region     = "${POWERVS_REGION}"
```

Comment:
Let's increase the processors for bootstrap and master and worker from 0.5 to 1.